tasks.go

```diff
@@ -36,7 +36,28 @@ func NewTask(upid UPID, client *Client) *Task {
 }
 
 func (t *Task) Ping() error {
-	return t.client.Get(fmt.Sprintf("/nodes/%s/tasks/%s/status", t.Node, t.UPID), t)
+	tmp := NewTask(t.UPID, t.client)
+	err := t.client.Get(fmt.Sprintf("/nodes/%s/tasks/%s/status", t.Node, t.UPID), t)
+	if nil != err || nil == t {
```

Comment:
how would you ever get here with t being nil? what were you testing that got you to have to do a tmp task and check if these things are nil?

Reply:
When the task has just been submitted and has not started to execute